wfs3/collection_meta_data.go

```diff
@@ -54,7 +54,7 @@ func CollectionsMetaData(p *data_provider.Provider, serveAddress string, checkOn
 
 	csInfo := CollectionsInfo{Links: []*Link{}, Collections: []*CollectionInfo{}}
 	for _, cn := range cNames {
-		collectionUrl := fmt.Sprintf("http://%v/collections/%v", serveAddress, cn)
+		collectionUrl := fmt.Sprintf("%v/collections/%v", serveAddress, cn)
```

Comment:
Per previous comment (go-wfs-config.toml) I don't think the protocol belongs in serveAddress.